coramin/__init__.py

```diff
@@ -1,3 +1,7 @@
+import sys
+if sys.version_info.major != 3 or sys.version_info.minor < 6:
+    raise EnvironmentError('Coramin only supports Python 3.6 and newer.')
+
```

Comment:
Is this the "standard" way to do this? Or should it be done (or also done) in setup.py?